--- na/10025	2017-03-17 18:39:13.000000000 +0800
+++ jp/10025	2017-03-17 18:39:13.000000000 +0800
@@ -184,9 +184,63 @@
     "initialType": "BALANCE",
     "magiaCharaNo": 100200,
     "magiaId": 10025,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201103,
+            "artId2": 390201203,
+            "groupId": 1085,
+            "id": 131601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "miniCharaNo": 100201,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201102,
+                "effectCode": "ATTACK",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201202,
+                "effectCode": "DEFENSE",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201102,
+            "artId2": 390201202,
+            "groupId": 1085,
+            "id": 131600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 900,
     "rateGainMpDef": 900,
