--- na/10045	2017-03-17 18:39:13.000000000 +0800
+++ jp/10045	2017-03-17 18:39:13.000000000 +0800
@@ -196,9 +196,311 @@
     "initialType": "DEFENSE",
     "magiaCharaNo": 100400,
     "magiaId": 10045,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690209803,
+                "effectCode": "RESTRAINT",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 200,
+                "targetId": "SELF",
+                "verbCode": "ENCHANT"
+            },
+            "art2": {
+                "artId": 390201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690209803,
+            "artId2": 390201105,
+            "eventArt1": {
+                "artId": 1126004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1126_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1126004,
+            "eventDescription": "＆「絡まりリボン」追加獲得数＋15",
+            "eventId": 1126,
+            "groupId": 1147,
+            "id": 160401,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206106,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206205,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690206106,
+            "artId2": 690206205,
+            "groupId": 1119,
+            "id": 178601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510004104,
+                "effectCode": "STUN",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 375,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 420002708,
+                "effectCode": "BLAST",
+                "effectValue": 225,
+                "enableTurn": 2,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art3": {
+                "artId": 420002004,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 2,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510004104,
+            "artId2": 420002708,
+            "artId3": 420002004,
+            "eventArt1": {
+                "artId": 1040004,
+                "effectCode": "DROPADD",
+                "effectValue": 7000,
+                "genericValue": "EVENT_DAILYTOWER_1040_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1040004,
+            "eventDescription": "＆回数券＋7",
+            "eventId": 1040,
+            "groupId": 1104,
+            "id": 127001,
+            "intervalTurn": 8,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201106,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201205,
+                "effectCode": "DEFENSE",
+                "effectValue": 375,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 330201106,
+            "artId2": 330201205,
+            "groupId": 1085,
+            "id": 106711,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "regularEventArt1": {
+                "artId": 350201505,
+                "effectCode": "RESIST",
+                "effectValue": 400,
+                "enableTurn": 5,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "regularEventArtId1": 350201505,
+            "regularEventDescription": "【殲滅戦時追加で状態異常耐性UP[Ⅴ](自/5T)】",
+            "regularEventId": 2011,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 100401,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690209802,
+                "effectCode": "RESTRAINT",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 150,
+                "targetId": "SELF",
+                "verbCode": "ENCHANT"
+            },
+            "art2": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690209802,
+            "artId2": 390201104,
+            "eventArt1": {
+                "artId": 1126003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1126_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1126003,
+            "eventDescription": "＆「絡まりリボン」追加獲得数＋5",
+            "eventId": 1126,
+            "groupId": 1147,
+            "id": 160400,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206105,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206204,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690206105,
+            "artId2": 690206204,
+            "groupId": 1119,
+            "id": 178600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510004102,
+                "effectCode": "STUN",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 325,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 420002707,
+                "effectCode": "BLAST",
+                "effectValue": 200,
+                "enableTurn": 2,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art3": {
+                "artId": 420002003,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "enableTurn": 2,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510004102,
+            "artId2": 420002707,
+            "artId3": 420002003,
+            "eventArt1": {
+                "artId": 1040003,
+                "effectCode": "DROPADD",
+                "effectValue": 3000,
+                "genericValue": "EVENT_DAILYTOWER_1040_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1040003,
+            "eventDescription": "＆回数券＋3",
+            "eventId": 1040,
+            "groupId": 1104,
+            "id": 127000,
+            "intervalTurn": 10,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 330201105,
+            "artId2": 330201204,
+            "groupId": 1085,
+            "id": 106710,
+            "intervalTurn": 9,
+            "messageId": 70,
+            "regularEventArt1": {
+                "artId": 350201503,
+                "effectCode": "RESIST",
+                "effectValue": 300,
+                "enableTurn": 5,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "regularEventArtId1": 350201503,
+            "regularEventDescription": "【殲滅戦時追加で状態異常耐性UP[Ⅲ](自/5T)】",
+            "regularEventId": 2011,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "randomDoppelCardMagiaMap": {
         "10040": {
             "art1": {
