--- na/10055	2017-03-17 18:39:13.000000000 +0800
+++ jp/10055	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 100000107,
-            "effectValue": 7200,
+            "effectValue": 11520,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -147,7 +147,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190100581,
-            "effectValue": 16720,
+            "effectValue": 26752,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -193,9 +193,63 @@
     "initialType": "ATTACK",
     "magiaCharaNo": 100500,
     "magiaId": 10055,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201103,
+            "artId2": 390201203,
+            "groupId": 1085,
+            "id": 131601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "miniCharaNo": 100501,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201102,
+                "effectCode": "ATTACK",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201202,
+                "effectCode": "DEFENSE",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201102,
+            "artId2": 390201202,
+            "groupId": 1085,
+            "id": 131600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1000,
     "rateGainMpDef": 800,
