--- na/10075	2017-03-17 18:39:13.000000000 +0800
+++ jp/10075	2017-03-17 18:39:13.000000000 +0800
@@ -202,9 +202,289 @@
     "initialType": "MAGIA",
     "magiaCharaNo": 100700,
     "magiaId": 10075,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690207705,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 60,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 390201404,
+                "effectCode": "MAGIA",
+                "effectValue": 125,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690207705,
+            "artId2": 390201404,
+            "groupId": 1129,
+            "id": 119201,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201303,
+                "effectCode": "DAMAGE",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 390201903,
+                "effectCode": "MP_GAIN",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201303,
+            "artId2": 390201603,
+            "artId3": 390201903,
+            "eventArt1": {
+                "artId": 1136004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1136_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1136004,
+            "eventDescription": "＆「キレイな川魚」追加獲得数＋15",
+            "eventId": 1136,
+            "groupId": 1087,
+            "id": 163401,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410002106,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201407,
+                "effectCode": "MAGIA",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410002106,
+            "artId2": 310201407,
+            "groupId": 1095,
+            "id": 156401,
+            "intervalTurn": 9,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201606,
+                "effectCode": "ACCEL",
+                "effectValue": 225,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 630206104,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201606,
+            "artId2": 630206104,
+            "eventArt1": {
+                "artId": 1127004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1127_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1127004,
+            "eventDescription": "＆「アクアリウムへの招待状」追加獲得数＋15",
+            "eventId": 1127,
+            "groupId": 1090,
+            "id": 152901,
+            "intervalTurn": 9,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 100700,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690207704,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 50,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 390201403,
+                "effectCode": "MAGIA",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690207704,
+            "artId2": 390201403,
+            "groupId": 1129,
+            "id": 119200,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201302,
+                "effectCode": "DAMAGE",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201602,
+                "effectCode": "ACCEL",
+                "effectValue": 125,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 390201902,
+                "effectCode": "MP_GAIN",
+                "effectValue": 75,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201302,
+            "artId2": 390201602,
+            "artId3": 390201902,
+            "eventArt1": {
+                "artId": 1136003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1136_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1136003,
+            "eventDescription": "＆「キレイな川魚」追加獲得数＋5",
+            "eventId": 1136,
+            "groupId": 1087,
+            "id": 163400,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410002105,
+                "effectCode": "DEFENSE",
+                "effectValue": 250,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201406,
+                "effectCode": "MAGIA",
+                "effectValue": 175,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410002105,
+            "artId2": 310201406,
+            "groupId": 1095,
+            "id": 156400,
+            "intervalTurn": 10,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201605,
+                "effectCode": "ACCEL",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 630206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201605,
+            "artId2": 630206103,
+            "eventArt1": {
+                "artId": 1127003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1127_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1127003,
+            "eventDescription": "＆「アクアリウムへの招待状」追加獲得数＋5",
+            "eventId": 1127,
+            "groupId": 1090,
+            "id": 152900,
+            "intervalTurn": 10,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1200,
     "rateGainMpDef": 1200,
