--- na/10125	2017-03-17 18:39:13.000000000 +0800
+++ jp/10125	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 100000107,
-            "effectValue": 7200,
+            "effectValue": 11520,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -158,7 +158,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190101281,
-            "effectValue": 16720,
+            "effectValue": 26752,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -204,9 +204,291 @@
     "initialType": "SUPPORT",
     "magiaCharaNo": 101200,
     "magiaId": 10125,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201503,
+                "effectCode": "RESIST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201104,
+            "artId2": 390201503,
+            "groupId": 1085,
+            "id": 132001,
+            "regularEventArt1": {
+                "artId": 690209000,
+                "effectCode": "DARKNESS",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "regularEventArtId1": 690209000,
+            "regularEventDescription": "【殲滅戦時追加で必ず暗闇無効】",
+            "regularEventId": 2016,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206105,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206204,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690206105,
+            "artId2": 690206204,
+            "groupId": 1119,
+            "id": 147201,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410002210,
+                "effectCode": "DAMAGE",
+                "effectValue": 500,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 610206210,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 500,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 410002210,
+            "artId2": 610206210,
+            "eventArt1": {
+                "artId": 1188004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1188_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1188004,
+            "eventDescription": "＆「ハワイ行のチケット」追加獲得数＋15",
+            "eventId": 1188,
+            "groupId": 1096,
+            "id": 174701,
+            "intervalTurn": 5,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 510303906,
+                "effectCode": "CHARM",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 425,
+                "targetId": "ALL",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410302006,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510303906,
+            "artId2": 410302006,
+            "eventArt1": {
+                "artId": 1100004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1100_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1100004,
+            "eventDescription": "＆「乙女の秘密」追加獲得数＋15",
+            "eventId": 1100,
+            "groupId": 1103,
+            "id": 150101,
+            "intervalTurn": 8,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 101200,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201502,
+                "effectCode": "RESIST",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201103,
+            "artId2": 390201502,
+            "groupId": 1085,
+            "id": 132000,
+            "regularEventArt1": {
+                "artId": 690209010,
+                "effectCode": "DARKNESS",
+                "growPoint": 0,
+                "probability": 700,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "regularEventArtId1": 690209010,
+            "regularEventDescription": "【殲滅戦時追加で確率で暗闇無効[Ⅹ]】",
+            "regularEventId": 2016,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206104,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206203,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690206104,
+            "artId2": 690206203,
+            "groupId": 1119,
+            "id": 147200,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410002208,
+                "effectCode": "DAMAGE",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 610206208,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 410002208,
+            "artId2": 610206208,
+            "eventArt1": {
+                "artId": 1188003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1188_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1188003,
+            "eventDescription": "＆「ハワイ行のチケット」追加獲得数＋5",
+            "eventId": 1188,
+            "groupId": 1096,
+            "id": 174700,
+            "intervalTurn": 6,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 510303905,
+                "effectCode": "CHARM",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 400,
+                "targetId": "ALL",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410302005,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510303905,
+            "artId2": 410302005,
+            "eventArt1": {
+                "artId": 1100003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1100_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1100003,
+            "eventDescription": "＆「乙女の秘密」追加獲得数＋5",
+            "eventId": 1100,
+            "groupId": 1103,
+            "id": 150100,
+            "intervalTurn": 9,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1200,
     "rateGainMpDef": 1000,
