--- na/10155	2017-03-17 18:39:13.000000000 +0800
+++ jp/10155	2017-03-17 18:39:13.000000000 +0800
@@ -76,7 +76,7 @@
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 200316402,
+            "artId": 200316405,
             "effectCode": "MP_DAMAGE",
             "effectValue": 350,
             "growPoint": 0,
@@ -102,7 +102,7 @@
             "verbCode": "HEAL"
         },
         "artId1": 100300108,
-        "artId2": 200316402,
+        "artId2": 200316405,
         "artId3": 510305000,
         "artId4": 200200503,
         "groupId": 1014,
@@ -163,7 +163,7 @@
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 200316407,
+            "artId": 200316410,
             "effectCode": "MP_DAMAGE",
             "effectValue": 600,
             "growPoint": 0,
@@ -189,7 +189,7 @@
             "verbCode": "HEAL"
         },
         "artId1": 190404581,
-        "artId2": 200316407,
+        "artId2": 200316410,
         "artId3": 510305000,
         "artId4": 200200505,
         "groupId": 1014,
