--- na/10154	2017-03-17 18:39:13.000000000 +0800
+++ jp/10154	2017-03-17 18:39:13.000000000 +0800
@@ -76,7 +76,7 @@
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 200316401,
+            "artId": 200316404,
             "effectCode": "MP_DAMAGE",
             "effectValue": 300,
             "growPoint": 0,
@@ -94,7 +94,7 @@
             "verbCode": "CONDITION_BAD"
         },
         "artId1": 100300106,
-        "artId2": 200316401,
+        "artId2": 200316404,
         "artId3": 510305000,
         "groupId": 1014,
         "id": 10154,
