--- na/10165	2017-03-17 18:39:13.000000000 +0800
+++ jp/10165	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 101200107,
-            "effectValue": 1515,
+            "effectValue": 2376,
             "growPoint": 80,
             "targetId": "RANDOM5",
             "verbCode": "ATTACK"
@@ -138,7 +138,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190101681,
-            "effectValue": 3310,
+            "effectValue": 5518,
             "growPoint": 176,
             "targetId": "RANDOM5",
             "verbCode": "ATTACK"
