--- na/10165	2017-03-17 18:39:13.000000000 +0800
+++ jp/10165	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 101200107,
-            "effectValue": 1515,
+            "effectValue": 2376,
             "growPoint": 80,
             "targetId": "RANDOM5",
             "verbCode": "ATTACK"
@@ -138,7 +138,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190101681,
-            "effectValue": 3310,
+            "effectValue": 5518,
             "growPoint": 176,
             "targetId": "RANDOM5",
             "verbCode": "ATTACK"
@@ -175,9 +175,279 @@
     "initialType": "ATTACK",
     "magiaCharaNo": 101600,
     "magiaId": 10165,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690206106,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690209200,
+                "effectCode": "BAN_SKILL",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206106,
+            "artId2": 690209200,
+            "groupId": 1119,
+            "id": 184301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206304,
+                "effectCode": "CRITICAL",
+                "effectValue": 2000,
+                "growPoint": 0,
+                "probability": 250,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 390201102,
+                "effectCode": "ATTACK",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208300,
+                "effectCode": "CHARM",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206304,
+            "artId2": 390201102,
+            "artId3": 690208300,
+            "groupId": 1121,
+            "id": 144901,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 310201810,
+                "effectCode": "BLAST",
+                "effectValue": 600,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 310201609,
+                "effectCode": "ACCEL",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 310201810,
+            "artId2": 310201609,
+            "groupId": 1092,
+            "id": 184901,
+            "intervalTurn": 5,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206203,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690207703,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 40,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 390201104,
+            "artId2": 690206203,
+            "artId3": 690207703,
+            "eventArt1": {
+                "artId": 1121004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1121_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1121004,
+            "eventDescription": "＆「大凧」追加獲得数＋15",
+            "eventId": 1121,
+            "groupId": 1085,
+            "id": 158301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "miniCharaNo": 101600,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690206105,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690209200,
+                "effectCode": "BAN_SKILL",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206105,
+            "artId2": 690209200,
+            "groupId": 1119,
+            "id": 184300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206303,
+                "effectCode": "CRITICAL",
+                "effectValue": 2000,
+                "growPoint": 0,
+                "probability": 200,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 390201101,
+                "effectCode": "ATTACK",
+                "effectValue": 50,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208300,
+                "effectCode": "CHARM",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206303,
+            "artId2": 390201101,
+            "artId3": 690208300,
+            "groupId": 1121,
+            "id": 144900,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 310201809,
+                "effectCode": "BLAST",
+                "effectValue": 550,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 310201608,
+                "effectCode": "ACCEL",
+                "effectValue": 275,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 310201809,
+            "artId2": 310201608,
+            "groupId": 1092,
+            "id": 184900,
+            "intervalTurn": 6,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206202,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690207702,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 30,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 390201103,
+            "artId2": 690206202,
+            "artId3": 690207702,
+            "eventArt1": {
+                "artId": 1121003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1121_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1121003,
+            "eventDescription": "＆「大凧」追加獲得数＋5",
+            "eventId": 1121,
+            "groupId": 1085,
+            "id": 158300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1000,
     "rateGainMpDef": 800,
