--- na/10185	2017-03-17 18:39:13.000000000 +0800
+++ jp/10185	2017-03-17 18:39:13.000000000 +0800
@@ -167,9 +167,307 @@
     "initialType": "BALANCE",
     "magiaCharaNo": 101800,
     "magiaId": 10185,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201804,
+                "effectCode": "BLAST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208500,
+                "effectCode": "RESTRAINT",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201804,
+            "artId2": 390201603,
+            "artId3": 690208500,
+            "eventArt1": {
+                "artId": 1121002,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1121_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1121002,
+            "eventDescription": "＆「大凧」追加獲得数＋5",
+            "eventId": 1121,
+            "groupId": 1092,
+            "id": 129301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201106,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201502,
+                "effectCode": "RESIST",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201106,
+            "artId2": 390201502,
+            "groupId": 1085,
+            "id": 175201,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 630206305,
+                "effectCode": "CRITICAL",
+                "effectValue": 2000,
+                "enableTurn": 3,
+                "growPoint": 0,
+                "probability": 300,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201105,
+            "artId2": 330201204,
+            "artId3": 630206305,
+            "groupId": 1085,
+            "id": 126101,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302408,
+                "effectCode": "RESIST",
+                "effectValue": 225,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 410302109,
+                "effectCode": "DEFENSE",
+                "effectValue": 450,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 410302408,
+            "artId2": 410302109,
+            "eventArt1": {
+                "artId": 1185004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1185_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1185004,
+            "eventDescription": "＆「マギウスの指令書」追加獲得数＋15",
+            "eventId": 1185,
+            "groupId": 1098,
+            "id": 186901,
+            "intervalTurn": 5,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 101800,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201802,
+                "effectCode": "BLAST",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201602,
+                "effectCode": "ACCEL",
+                "effectValue": 125,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208500,
+                "effectCode": "RESTRAINT",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201802,
+            "artId2": 390201602,
+            "artId3": 690208500,
+            "eventArt1": {
+                "artId": 1121001,
+                "effectCode": "DROPADD",
+                "effectValue": 1000,
+                "genericValue": "EVENT_DAILYTOWER_1121_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1121001,
+            "eventDescription": "＆「大凧」追加獲得数＋1",
+            "eventId": 1121,
+            "groupId": 1092,
+            "id": 129300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201501,
+                "effectCode": "RESIST",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201105,
+            "artId2": 390201501,
+            "groupId": 1085,
+            "id": 175200,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 330201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 630206304,
+                "effectCode": "CRITICAL",
+                "effectValue": 2000,
+                "enableTurn": 3,
+                "growPoint": 0,
+                "probability": 250,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201104,
+            "artId2": 330201203,
+            "artId3": 630206304,
+            "groupId": 1085,
+            "id": 126100,
+            "intervalTurn": 9,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302407,
+                "effectCode": "RESIST",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 410302108,
+                "effectCode": "DEFENSE",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 410302407,
+            "artId2": 410302108,
+            "eventArt1": {
+                "artId": 1185003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1185_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1185003,
+            "eventDescription": "＆「マギウスの指令書」追加獲得数＋5",
+            "eventId": 1185,
+            "groupId": 1098,
+            "id": 186900,
+            "intervalTurn": 6,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 900,
     "rateGainMpDef": 900,
