--- na/11085	2017-03-17 18:39:13.000000000 +0800
+++ jp/11085	2017-03-17 18:39:13.000000000 +0800
@@ -42,13 +42,13 @@
             "rankGift": "RANK_3"
         },
         "gift5": {
-            "id": 505,
+            "id": 508,
             "price": 5000,
             "rank": 3,
             "rankGift": "RANK_3"
         },
         "gift6": {
-            "id": 406,
+            "id": 407,
             "price": 3000,
             "rank": 2,
             "rankGift": "RANK_2"
@@ -57,8 +57,8 @@
         "giftId2": 132,
         "giftId3": 133,
         "giftId4": 506,
-        "giftId5": 505,
-        "giftId6": 406,
+        "giftId5": 508,
+        "giftId6": 407,
         "giftNum1": 3,
         "giftNum2": 15,
         "giftNum3": 10,
@@ -206,9 +206,249 @@
     "initialType": "ATTACK",
     "magiaCharaNo": 110800,
     "magiaId": 11085,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201805,
+                "effectCode": "BLAST",
+                "effectValue": 350,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201805,
+            "artId2": 390201104,
+            "groupId": 1092,
+            "id": 182601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206106,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690209200,
+                "effectCode": "BAN_SKILL",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206106,
+            "artId2": 690209200,
+            "groupId": 1119,
+            "id": 184301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 330201106,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 630206206,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201106,
+            "artId2": 630206206,
+            "groupId": 1085,
+            "id": 176601,
+            "intervalTurn": 7,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 630206107,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 350,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 630206807,
+                "effectCode": "AVOID",
+                "enableTurn": 3,
+                "growPoint": 0,
+                "probability": 400,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 630206107,
+            "artId2": 630206807,
+            "eventArt1": {
+                "artId": 1150004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1150_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1150004,
+            "eventDescription": "＆「サークルリスト」追加獲得数＋15",
+            "eventId": 1150,
+            "groupId": 1119,
+            "id": 169601,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 110800,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201804,
+                "effectCode": "BLAST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201804,
+            "artId2": 390201103,
+            "groupId": 1092,
+            "id": 182600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690206105,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690209200,
+                "effectCode": "BAN_SKILL",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206105,
+            "artId2": 690209200,
+            "groupId": 1119,
+            "id": 184300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 630206205,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201105,
+            "artId2": 630206205,
+            "groupId": 1085,
+            "id": 176600,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 630206106,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 630206806,
+                "effectCode": "AVOID",
+                "enableTurn": 3,
+                "growPoint": 0,
+                "probability": 350,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 630206106,
+            "artId2": 630206806,
+            "eventArt1": {
+                "artId": 1150003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1150_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1150003,
+            "eventDescription": "＆「サークルリスト」追加獲得数＋5",
+            "eventId": 1150,
+            "groupId": 1119,
+            "id": 169600,
+            "intervalTurn": 10,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1200,
     "rateGainMpDef": 1200,
