--- na/13015	2017-03-17 18:39:13.000000000 +0800
+++ jp/13015	2017-03-17 18:39:13.000000000 +0800
@@ -240,6 +240,7 @@
             "artId3": 600216700,
             "groupId": 1086,
             "id": 13012,
+            "skillDisplayType": "EXTRA",
             "type": "STARTUP"
         }
     ],
@@ -281,6 +282,7 @@
             "artId3": 600216700,
             "groupId": 1086,
             "id": 13011,
+            "skillDisplayType": "EXTRA",
             "type": "STARTUP"
         }
     ],
