--- na/20055	2017-03-17 18:39:13.000000000 +0800
+++ jp/20055	2017-03-17 18:39:13.000000000 +0800
@@ -95,15 +95,6 @@
             "verbCode": "CONDITION_BAD"
         },
         "art4": {
-            "artId": 610307800,
-            "effectCode": "SKILL_QUICK",
-            "enableTurn": 1,
-            "growPoint": 0,
-            "probability": 1000,
-            "targetId": "ALL",
-            "verbCode": "CONDITION_GOOD"
-        },
-        "art5": {
             "artId": 9011,
             "effectCode": "DEFENSE",
             "effectValue": 500,
@@ -115,8 +106,7 @@
         "artId1": 100300108,
         "artId2": 510304310,
         "artId3": 530304710,
-        "artId4": 610307800,
-        "artId5": 9011,
+        "artId4": 9011,
         "groupId": 1014,
         "id": 20055,
     },
@@ -193,19 +183,9 @@
             "targetId": "ALL",
             "verbCode": "CONDITION_BAD"
         },
-        "art4": {
-            "artId": 610307800,
-            "effectCode": "SKILL_QUICK",
-            "enableTurn": 1,
-            "growPoint": 0,
-            "probability": 1000,
-            "targetId": "ALL",
-            "verbCode": "CONDITION_GOOD"
-        },
         "artId1": 190200581,
         "artId2": 510304311,
         "artId3": 530304711,
-        "artId4": 610307800,
         "groupId": 1014,
         "id": 20058,
     },
