--- na/20054	2017-03-17 18:39:13.000000000 +0800
+++ jp/20054	2017-03-17 18:39:13.000000000 +0800
@@ -85,15 +85,6 @@
             "verbCode": "CONDITION_BAD"
         },
         "art3": {
-            "artId": 610307800,
-            "effectCode": "SKILL_QUICK",
-            "enableTurn": 1,
-            "growPoint": 0,
-            "probability": 1000,
-            "targetId": "ALL",
-            "verbCode": "CONDITION_GOOD"
-        },
-        "art4": {
             "artId": 9011,
             "effectCode": "DEFENSE",
             "effectValue": 500,
@@ -104,8 +95,7 @@
         },
         "artId1": 100300106,
         "artId2": 510304308,
-        "artId3": 610307800,
-        "artId4": 9011,
+        "artId3": 9011,
         "groupId": 1014,
         "id": 20054,
     },
