--- na/20075	2017-03-17 18:39:13.000000000 +0800
+++ jp/20075	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 101100108,
-            "effectValue": 1947,
+            "effectValue": 3052,
             "growPoint": 100,
             "targetId": "RANDOM4",
             "verbCode": "ATTACK"
@@ -149,7 +149,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190200781,
-            "effectValue": 4513,
+            "effectValue": 7077,
             "growPoint": 220,
             "targetId": "RANDOM4",
             "verbCode": "ATTACK"
