--- na/21005	2017-03-17 18:39:13.000000000 +0800
+++ jp/21005	2017-03-17 18:39:13.000000000 +0800
@@ -192,9 +192,285 @@
     "initialType": "SUPPORT",
     "magiaCharaNo": 210000,
     "magiaId": 21005,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690207703,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 40,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206104,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690208400,
+                "effectCode": "STUN",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690207703,
+            "artId2": 690206104,
+            "artId3": 690208400,
+            "groupId": 1129,
+            "id": 139701,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390203209,
+                "effectCode": "DEFENSE",
+                "effectValue": 900,
+                "growPoint": 20,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF_HPMAX"
+            },
+            "art2": {
+                "artId": 690206204,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 390203209,
+            "artId2": 690206204,
+            "groupId": 1086,
+            "id": 151501,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410302109,
+                "effectCode": "DEFENSE",
+                "effectValue": 450,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201807,
+                "effectCode": "BLAST",
+                "effectValue": 450,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302109,
+            "artId2": 310201807,
+            "eventArt1": {
+                "artId": 1126004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1126_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1126004,
+            "eventDescription": "＆「絡まりリボン」追加獲得数＋15",
+            "eventId": 1126,
+            "groupId": 1095,
+            "id": 160501,
+            "intervalTurn": 6,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302405,
+                "effectCode": "RESIST",
+                "effectValue": 150,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201808,
+                "effectCode": "BLAST",
+                "effectValue": 500,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302405,
+            "artId2": 310201808,
+            "eventArt1": {
+                "artId": 1045004,
+                "effectCode": "DROPADD",
+                "effectValue": 7000,
+                "genericValue": "EVENT_DAILYTOWER_1045_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1045004,
+            "eventDescription": "＆「ブラックカード」入手時＋7",
+            "eventId": 1045,
+            "groupId": 1092,
+            "id": 128801,
+            "intervalTurn": 6,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 210000,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690207702,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 30,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690208400,
+                "effectCode": "STUN",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690207702,
+            "artId2": 690206103,
+            "artId3": 690208400,
+            "groupId": 1129,
+            "id": 139700,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390203207,
+                "effectCode": "DEFENSE",
+                "effectValue": 700,
+                "growPoint": 20,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF_HPMAX"
+            },
+            "art2": {
+                "artId": 690206203,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 390203207,
+            "artId2": 690206203,
+            "groupId": 1086,
+            "id": 151500,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 410302108,
+                "effectCode": "DEFENSE",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201806,
+                "effectCode": "BLAST",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302108,
+            "artId2": 310201806,
+            "eventArt1": {
+                "artId": 1126003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1126_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1126003,
+            "eventDescription": "＆「絡まりリボン」追加獲得数＋5",
+            "eventId": 1126,
+            "groupId": 1095,
+            "id": 160500,
+            "intervalTurn": 7,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302404,
+                "effectCode": "RESIST",
+                "effectValue": 125,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201807,
+                "effectCode": "BLAST",
+                "effectValue": 450,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302404,
+            "artId2": 310201807,
+            "eventArt1": {
+                "artId": 1045003,
+                "effectCode": "DROPADD",
+                "effectValue": 3000,
+                "genericValue": "EVENT_DAILYTOWER_1045_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1045003,
+            "eventDescription": "＆「ブラックカード」入手時＋3",
+            "eventId": 1045,
+            "groupId": 1092,
+            "id": 128800,
+            "intervalTurn": 7,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1200,
     "rateGainMpDef": 1000,
