--- na/30022	2017-03-17 18:39:13.000000000 +0800
+++ jp/30022	2017-03-17 18:39:13.000000000 +0800
@@ -71,32 +71,32 @@
     "cardMagia": {
         "art1": {
             "artId": 100000101,
-            "effectValue": 6000,
+            "effectValue": 9600,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 330301202,
-            "effectCode": "DEFENSE",
-            "effectValue": 263,
+            "artId": 330301102,
+            "effectCode": "ATTACK",
+            "effectValue": 175,
             "enableTurn": 3,
             "growPoint": 25,
             "targetId": "ALL",
             "verbCode": "BUFF"
         },
         "art3": {
-            "artId": 330301102,
-            "effectCode": "ATTACK",
-            "effectValue": 175,
+            "artId": 330301202,
+            "effectCode": "DEFENSE",
+            "effectValue": 263,
             "enableTurn": 3,
             "growPoint": 25,
             "targetId": "ALL",
             "verbCode": "BUFF"
         },
         "artId1": 100000101,
-        "artId2": 330301202,
-        "artId3": 330301102,
+        "artId2": 330301102,
+        "artId3": 330301202,
         "groupId": 1012,
         "id": 30022,
     },
