--- na/30043	2017-03-17 18:39:13.000000000 +0800
+++ jp/30043	2017-03-17 18:39:13.000000000 +0800
@@ -77,6 +77,15 @@
             "verbCode": "ATTACK"
         },
         "art2": {
+            "artId": 330301204,
+            "effectCode": "DEFENSE",
+            "effectValue": 338,
+            "enableTurn": 3,
+            "growPoint": 25,
+            "targetId": "ALL",
+            "verbCode": "BUFF"
+        },
+        "art3": {
             "artId": 610207600,
             "effectCode": "PROVOKE",
             "enableTurn": 1,
@@ -85,18 +94,9 @@
             "targetId": "SELF",
             "verbCode": "CONDITION_GOOD"
         },
-        "art3": {
-            "artId": 310201204,
-            "effectCode": "DEFENSE",
-            "effectValue": 300,
-            "enableTurn": 1,
-            "growPoint": 25,
-            "targetId": "SELF",
-            "verbCode": "BUFF"
-        },
         "artId1": 100300103,
-        "artId2": 610207600,
-        "artId3": 310201204,
+        "artId2": 330301204,
+        "artId3": 610207600,
         "groupId": 1014,
         "id": 30043,
     },
