--- na/30302	2017-03-17 18:39:13.000000000 +0800
+++ jp/30302	2017-03-17 18:39:13.000000000 +0800
@@ -71,7 +71,7 @@
     "cardMagia": {
         "art1": {
             "artId": 100600101,
-            "effectValue": 4500,
+            "effectValue": 4815,
             "growPoint": 100,
             "targetId": "HORIZONTAL",
             "verbCode": "ATTACK"
@@ -91,6 +91,16 @@
     },
     "cardSkill": {
         "art1": {
+            "artId": 630407703,
+            "effectCode": "AUTO_HEAL",
+            "effectValue": 40,
+            "enableTurn": 3,
+            "growPoint": 10,
+            "probability": 1000,
+            "targetId": "CONNECT",
+            "verbCode": "CONDITION_GOOD"
+        },
+        "art2": {
             "artId": 610410700,
             "effectCode": "FOG",
             "effectValue": 250,
@@ -100,7 +110,7 @@
             "targetId": "CONNECT",
             "verbCode": "ENCHANT"
         },
-        "art2": {
+        "art3": {
             "artId": 610410100,
             "effectCode": "POISON",
             "effectValue": 50,
@@ -110,9 +120,10 @@
             "targetId": "CONNECT",
             "verbCode": "ENCHANT"
         },
-        "artId1": 610410700,
-        "artId2": 610410100,
-        "groupId": 1151,
+        "artId1": 630407703,
+        "artId2": 610410700,
+        "artId3": 610410100,
+        "groupId": 1129,
         "id": 30302,
     },
     "charaNo": 3030,
