--- na/30325	2017-03-17 18:39:13.000000000 +0800
+++ jp/30325	2017-03-17 18:39:13.000000000 +0800
@@ -188,9 +188,365 @@
     "initialType": "MAGIA",
     "magiaCharaNo": 303200,
     "magiaId": 30325,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201407,
+                "effectCode": "MAGIA",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690209300,
+                "effectCode": "BAN_MAGIA",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201407,
+            "artId2": 690209300,
+            "eventArt1": {
+                "artId": 1111004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1111_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1111004,
+            "eventDescription": "＆「いたずら駄菓子」追加獲得数＋15",
+            "eventId": 1111,
+            "groupId": 1088,
+            "id": 154501,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201604,
+                "effectCode": "ACCEL",
+                "effectValue": 175,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201903,
+                "effectCode": "MP_GAIN",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 390201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201604,
+            "artId2": 390201903,
+            "artId3": 390201204,
+            "eventArt1": {
+                "artId": 1154004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1154_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1154004,
+            "eventDescription": "＆「錆びついた紋章」追加獲得数＋15",
+            "eventId": 1154,
+            "groupId": 1090,
+            "id": 171201,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 610206106,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 610214606,
+                "effectCode": "DAMAGE_UP_BAD",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 310201404,
+                "effectCode": "MAGIA",
+                "effectValue": 125,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 610206106,
+            "artId2": 610214606,
+            "artId3": 310201404,
+            "eventArt1": {
+                "artId": 1139004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_BRANCH_1139_PART_1",
+                "verbCode": "OTHER"
+            },
+            "eventArt2": {
+                "artId": 1139008,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_BRANCH_1139_PART_2",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1139004,
+            "eventArtId2": 1139008,
+            "eventDescription": "＆「貝殻の腕輪」「翡翠の腕輪」追加獲得数＋15",
+            "eventId": 1139,
+            "groupId": 1119,
+            "id": 164501,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302406,
+                "effectCode": "RESIST",
+                "effectValue": 175,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201407,
+                "effectCode": "MAGIA",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302406,
+            "artId2": 310201407,
+            "eventArt1": {
+                "artId": 1129004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_SINGLERAID_1129_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1129004,
+            "eventDescription": "＆「興味のハート」追加獲得数＋15",
+            "eventId": 1129,
+            "groupId": 1098,
+            "id": 160901,
+            "intervalTurn": 7,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 303200,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201406,
+                "effectCode": "MAGIA",
+                "effectValue": 175,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690209309,
+                "effectCode": "BAN_MAGIA",
+                "growPoint": 0,
+                "probability": 650,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201406,
+            "artId2": 690209309,
+            "eventArt1": {
+                "artId": 1111003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1111_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1111003,
+            "eventDescription": "＆「いたずら駄菓子」追加獲得数＋5",
+            "eventId": 1111,
+            "groupId": 1088,
+            "id": 154500,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201902,
+                "effectCode": "MP_GAIN",
+                "effectValue": 75,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 390201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201603,
+            "artId2": 390201902,
+            "artId3": 390201203,
+            "eventArt1": {
+                "artId": 1154003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1154_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1154003,
+            "eventDescription": "＆「錆びついた紋章」追加獲得数＋5",
+            "eventId": 1154,
+            "groupId": 1090,
+            "id": 171200,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 610206104,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 610214604,
+                "effectCode": "DAMAGE_UP_BAD",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 310201402,
+                "effectCode": "MAGIA",
+                "effectValue": 75,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 610206104,
+            "artId2": 610214604,
+            "artId3": 310201402,
+            "eventArt1": {
+                "artId": 1139003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_BRANCH_1139_PART_1",
+                "verbCode": "OTHER"
+            },
+            "eventArt2": {
+                "artId": 1139007,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_BRANCH_1139_PART_2",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1139003,
+            "eventArtId2": 1139007,
+            "eventDescription": "＆「貝殻の腕輪」「翡翠の腕輪」追加獲得数＋5",
+            "eventId": 1139,
+            "groupId": 1119,
+            "id": 164500,
+            "intervalTurn": 9,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 410302406,
+                "effectCode": "RESIST",
+                "effectValue": 175,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 310201406,
+                "effectCode": "MAGIA",
+                "effectValue": 175,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 410302406,
+            "artId2": 310201406,
+            "eventArt1": {
+                "artId": 1129003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_SINGLERAID_1129_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1129003,
+            "eventDescription": "＆「興味のハート」追加獲得数＋5",
+            "eventId": 1129,
+            "groupId": 1098,
+            "id": 160900,
+            "intervalTurn": 8,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1200,
     "rateGainMpDef": 1200,
