--- na/30435	2017-03-17 18:39:13.000000000 +0800
+++ jp/30435	2017-03-17 18:39:13.000000000 +0800
@@ -71,7 +71,7 @@
         "art1": {
             "artId": 100000207,
             "effectCode": "ALIGNMENT",
-            "effectValue": 5800,
+            "effectValue": 9280,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -160,7 +160,7 @@
         "art1": {
             "artId": 190304381,
             "effectCode": "ALIGNMENT",
-            "effectValue": 13640,
+            "effectValue": 21824,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -206,9 +206,311 @@
     "initialType": "BALANCE",
     "magiaCharaNo": 304300,
     "magiaId": 30435,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201804,
+                "effectCode": "BLAST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208300,
+                "effectCode": "CHARM",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201804,
+            "artId2": 390201603,
+            "artId3": 690208300,
+            "eventArt1": {
+                "artId": 1098004,
+                "effectCode": "DROPADD",
+                "effectValue": 7000,
+                "genericValue": "EVENT_DAILYTOWER_1098_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1098004,
+            "eventDescription": "＆「始まりの焚火」追加獲得数＋7",
+            "eventId": 1098,
+            "groupId": 1092,
+            "id": 138101,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201205,
+                "effectCode": "DEFENSE",
+                "effectValue": 375,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201105,
+            "artId2": 390201205,
+            "groupId": 1085,
+            "id": 154101,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 430302105,
+                "effectCode": "DEFENSE",
+                "effectValue": 150,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 430302105,
+            "artId2": 330201105,
+            "eventArt1": {
+                "artId": 1147004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1147_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1147004,
+            "eventDescription": "＆「ジュゥべえメダル」追加獲得数＋15",
+            "eventId": 1147,
+            "groupId": 1095,
+            "id": 168101,
+            "intervalTurn": 8,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 630220800,
+                "effectCode": "CONDITION_BAD",
+                "enableTurn": 3,
+                "genericValue": "NONE",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "art2": {
+                "artId": 630207705,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 60,
+                "enableTurn": 3,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 630206205,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 630220800,
+            "artId2": 630207705,
+            "artId3": 630206205,
+            "groupId": 1231,
+            "id": 180001,
+            "intervalTurn": 7,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 304300,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201802,
+                "effectCode": "BLAST",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201602,
+                "effectCode": "ACCEL",
+                "effectValue": 125,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 690208300,
+                "effectCode": "CHARM",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 390201802,
+            "artId2": 390201602,
+            "artId3": 690208300,
+            "eventArt1": {
+                "artId": 1098003,
+                "effectCode": "DROPADD",
+                "effectValue": 3000,
+                "genericValue": "EVENT_DAILYTOWER_1098_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1098003,
+            "eventDescription": "＆「始まりの焚火」追加獲得数＋3",
+            "eventId": 1098,
+            "groupId": 1092,
+            "id": 138100,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201104,
+            "artId2": 390201204,
+            "groupId": 1085,
+            "id": 154100,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 430302104,
+                "effectCode": "DEFENSE",
+                "effectValue": 125,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "ALL",
+                "verbCode": "DEBUFF"
+            },
+            "art2": {
+                "artId": 330201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 430302104,
+            "artId2": 330201104,
+            "eventArt1": {
+                "artId": 1147003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1147_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1147003,
+            "eventDescription": "＆「ジュゥべえメダル」追加獲得数＋5",
+            "eventId": 1147,
+            "groupId": 1095,
+            "id": 168100,
+            "intervalTurn": 9,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 630220800,
+                "effectCode": "CONDITION_BAD",
+                "enableTurn": 3,
+                "genericValue": "NONE",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "art2": {
+                "artId": 630207704,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 50,
+                "enableTurn": 3,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 630206204,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 630220800,
+            "artId2": 630207704,
+            "artId3": 630206204,
+            "groupId": 1231,
+            "id": 180000,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 900,
     "rateGainMpDef": 900,
