--- na/30495	2017-03-17 18:39:13.000000000 +0800
+++ jp/30495	2017-03-17 18:39:13.000000000 +0800
@@ -69,8 +69,9 @@
     "cardId": 30495,
     "cardMagia": {
         "art1": {
-            "artId": 100000107,
-            "effectValue": 7200,
+            "artId": 100000207,
+            "effectCode": "ALIGNMENT",
+            "effectValue": 9280,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -85,38 +86,48 @@
             "verbCode": "BUFF"
         },
         "art3": {
-            "artId": 200200800,
+            "artId": 200300800,
             "effectCode": "DEBUFF",
             "growPoint": 0,
-            "targetId": "SELF",
+            "targetId": "ALL",
             "verbCode": "REVOKE"
         },
-        "artId1": 100000107,
+        "artId1": 100000207,
         "artId2": 330301507,
-        "artId3": 200200800,
+        "artId3": 200300800,
         "groupId": 1012,
         "id": 30495,
     },
     "cardSkill": {
         "art1": {
-            "artId": 310401108,
+            "artId": 330401509,
+            "effectCode": "RESIST",
+            "effectValue": 400,
+            "enableTurn": 3,
+            "growPoint": 25,
+            "targetId": "CONNECT",
+            "verbCode": "BUFF"
+        },
+        "art2": {
+            "artId": 310401109,
             "effectCode": "ATTACK",
-            "effectValue": 375,
+            "effectValue": 400,
             "enableTurn": 1,
             "growPoint": 25,
             "targetId": "CONNECT",
             "verbCode": "BUFF"
         },
-        "art2": {
+        "art3": {
             "artId": 200400800,
             "effectCode": "DEBUFF",
             "growPoint": 0,
             "targetId": "CONNECT",
             "verbCode": "REVOKE"
         },
-        "artId1": 310401108,
-        "artId2": 200400800,
-        "groupId": 1085,
+        "artId1": 330401509,
+        "artId2": 310401109,
+        "artId3": 200400800,
+        "groupId": 1089,
         "id": 30495,
     },
     "charaNo": 3049,
@@ -133,7 +144,8 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190304981,
-            "effectValue": 16720,
+            "effectCode": "ALIGNMENT",
+            "effectValue": 21824,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -167,9 +179,321 @@
     "initialType": "BALANCE",
     "magiaCharaNo": 304900,
     "magiaId": 30495,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 390201205,
+                "effectCode": "DEFENSE",
+                "effectValue": 375,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201104,
+            "artId2": 690206103,
+            "artId3": 390201205,
+            "eventArt1": {
+                "artId": 1181004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1181_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1181004,
+            "eventDescription": "＆「初夢祈願」追加獲得数＋15",
+            "eventId": 1181,
+            "groupId": 1085,
+            "id": 184601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201804,
+                "effectCode": "BLAST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201604,
+                "effectCode": "ACCEL",
+                "effectValue": 175,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201804,
+            "artId2": 390201604,
+            "eventArt1": {
+                "artId": 1158004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158004,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋15",
+            "eventId": 1158,
+            "groupId": 1092,
+            "id": 151001,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510006000,
+                "effectCode": "BAN_MAGIA",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410002007,
+                "effectCode": "ATTACK",
+                "effectValue": 350,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510006000,
+            "artId2": 410002007,
+            "eventArt1": {
+                "artId": 1092004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1092_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1092004,
+            "eventDescription": "＆「一度きりの時間」追加獲得数＋15",
+            "eventId": 1092,
+            "groupId": 1113,
+            "id": 147601,
+            "intervalTurn": 4,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 200200800,
+                "effectCode": "DEBUFF",
+                "growPoint": 0,
+                "targetId": "SELF",
+                "verbCode": "REVOKE"
+            },
+            "art2": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 200200800,
+            "artId2": 330201105,
+            "eventArt1": {
+                "artId": 1158004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158004,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋15",
+            "eventId": 1158,
+            "groupId": 1082,
+            "id": 150901,
+            "intervalTurn": 8,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 304900,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 390201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201103,
+            "artId2": 690206103,
+            "artId3": 390201204,
+            "eventArt1": {
+                "artId": 1181003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1181_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1181003,
+            "eventDescription": "＆「初夢祈願」追加獲得数＋5",
+            "eventId": 1181,
+            "groupId": 1085,
+            "id": 184600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201803,
+                "effectCode": "BLAST",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201803,
+            "artId2": 390201603,
+            "eventArt1": {
+                "artId": 1158003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158003,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋5",
+            "eventId": 1158,
+            "groupId": 1092,
+            "id": 151000,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510006000,
+                "effectCode": "BAN_MAGIA",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410002006,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510006000,
+            "artId2": 410002006,
+            "eventArt1": {
+                "artId": 1092003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1092_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1092003,
+            "eventDescription": "＆「一度きりの時間」追加獲得数＋5",
+            "eventId": 1092,
+            "groupId": 1113,
+            "id": 147600,
+            "intervalTurn": 5,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 200200800,
+                "effectCode": "DEBUFF",
+                "growPoint": 0,
+                "targetId": "SELF",
+                "verbCode": "REVOKE"
+            },
+            "art2": {
+                "artId": 330201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 200200800,
+            "artId2": 330201104,
+            "eventArt1": {
+                "artId": 1158003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158003,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋5",
+            "eventId": 1158,
+            "groupId": 1082,
+            "id": 150900,
+            "intervalTurn": 9,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 900,
     "rateGainMpDef": 900,
