--- na/30525	2017-03-17 18:39:13.000000000 +0800
+++ jp/30525	2017-03-17 18:39:13.000000000 +0800
@@ -1,5 +1,5 @@
 {
-    "attack": 2292,
+    "attack": 2504,
     "attributeId": "DARK",
     "cardCustomize": {
         "bonusCode1": "ATTACK",
@@ -8,9 +8,9 @@
         "bonusCode4": "ACCEL",
         "bonusCode5": "BLAST",
         "bonusCode6": "CHARGE",
-        "bonusNum1": 70,
-        "bonusNum2": 70,
-        "bonusNum3": 70,
+        "bonusNum1": 90,
+        "bonusNum2": 50,
+        "bonusNum3": 60,
         "bonusNum4": 50,
         "bonusNum5": 90,
         "bonusNum6": 50,
@@ -36,19 +36,19 @@
             "rankGift": "RANK_3"
         },
         "gift4": {
-            "id": 553,
+            "id": 557,
             "price": 5000,
             "rank": 3,
             "rankGift": "RANK_3"
         },
         "gift5": {
-            "id": 503,
-            "price": 5000,
-            "rank": 3,
-            "rankGift": "RANK_3"
+            "id": 304,
+            "price": 1000,
+            "rank": 1,
+            "rankGift": "RANK_1"
         },
         "gift6": {
-            "id": 306,
+            "id": 305,
             "price": 1000,
             "rank": 1,
             "rankGift": "RANK_1"
@@ -56,15 +56,15 @@
         "giftId1": 551,
         "giftId2": 152,
         "giftId3": 153,
-        "giftId4": 553,
-        "giftId5": 503,
-        "giftId6": 306,
+        "giftId4": 557,
+        "giftId5": 304,
+        "giftId6": 305,
         "giftNum1": 3,
         "giftNum2": 15,
         "giftNum3": 10,
-        "giftNum4": 8,
-        "giftNum5": 10,
-        "giftNum6": 15
+        "giftNum4": 10,
+        "giftNum5": 8,
+        "giftNum6": 3
     },
     "cardId": 30525,
     "cardMagia": {
@@ -76,28 +76,36 @@
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 9379,
-            "effectCode": "POISON",
-            "effectValue": 50,
-            "enableTurn": 2,
+            "artId": 510304000,
+            "effectCode": "CHARM",
+            "enableTurn": 1,
             "growPoint": 0,
             "probability": 1000,
             "targetId": "ALL",
             "verbCode": "CONDITION_BAD"
         },
         "art3": {
-            "artId": 9356,
-            "effectCode": "BLINDNESS",
+            "artId": 330201808,
+            "effectCode": "BLAST",
             "effectValue": 500,
-            "enableTurn": 2,
+            "enableTurn": 3,
+            "growPoint": 25,
+            "targetId": "SELF",
+            "verbCode": "BUFF"
+        },
+        "art4": {
+            "artId": 630216515,
+            "effectCode": "DAMAGE_DOWN",
+            "enableTurn": 3,
             "growPoint": 0,
             "probability": 1000,
-            "targetId": "ALL",
-            "verbCode": "CONDITION_BAD"
+            "targetId": "SELF",
+            "verbCode": "IGNORE"
         },
         "artId1": 100300108,
-        "artId2": 9379,
-        "artId3": 9356,
+        "artId2": 510304000,
+        "artId3": 330201808,
+        "artId4": 630216515,
         "groupId": 1014,
         "id": 30525,
     },
@@ -121,9 +129,8 @@
             "verbCode": "BUFF"
         },
         "art3": {
-            "artId": 610411100,
-            "effectCode": "BLINDNESS",
-            "effectValue": 500,
+            "artId": 610409500,
+            "effectCode": "CHARM",
             "enableTurn": 1,
             "growPoint": 0,
             "probability": 1000,
@@ -132,16 +139,16 @@
         },
         "artId1": 310401809,
         "artId2": 310401109,
-        "artId3": 610411100,
+        "artId3": 610409500,
         "groupId": 1092,
         "id": 30525,
     },
     "charaNo": 3052,
     "commandType1": "MPUP",
-    "commandType2": "RANGE_H",
-    "commandType3": "RANGE_H",
-    "commandType4": "RANGE_V",
-    "commandType5": "RANGE_V",
+    "commandType2": "RANGE_V",
+    "commandType3": "RANGE_V",
+    "commandType4": "RANGE_H",
+    "commandType5": "RANGE_H",
     "defense": 2127,
     "doppel": {
         "archive": true,
@@ -149,35 +156,54 @@
     },
     "doppelCardMagia": {
         "art1": {
-            "artId": 100300110,
-            "effectValue": 3900,
-            "growPoint": 100,
+            "artId": 190305281,
+            "effectValue": 8360,
+            "growPoint": 220,
             "targetId": "ALL",
             "verbCode": "ATTACK"
         },
         "art2": {
-            "artId": 530304600,
-            "effectCode": "POISON",
-            "effectValue": 50,
-            "enableTurn": 3,
+            "artId": 510304000,
+            "effectCode": "CHARM",
+            "enableTurn": 1,
             "growPoint": 0,
             "probability": 1000,
             "targetId": "ALL",
             "verbCode": "CONDITION_BAD"
         },
         "art3": {
-            "artId": 530305600,
-            "effectCode": "BLINDNESS",
-            "effectValue": 500,
+            "artId": 350301809,
+            "effectCode": "BLAST",
+            "effectValue": 550,
+            "enableTurn": 5,
+            "growPoint": 25,
+            "targetId": "ALL",
+            "verbCode": "BUFF"
+        },
+        "art4": {
+            "artId": 650216515,
+            "effectCode": "DAMAGE_DOWN",
+            "enableTurn": 5,
+            "growPoint": 0,
+            "probability": 1000,
+            "targetId": "SELF",
+            "verbCode": "IGNORE"
+        },
+        "art5": {
+            "artId": 630220005,
+            "effectCode": "MP_PLUS_BLAST",
+            "effectValue": 50,
             "enableTurn": 3,
             "growPoint": 0,
             "probability": 1000,
-            "targetId": "ALL",
-            "verbCode": "CONDITION_BAD"
+            "targetId": "SELF",
+            "verbCode": "CONDITION_GOOD"
         },
-        "artId1": 100300110,
-        "artId2": 530304600,
-        "artId3": 530305600,
+        "artId1": 190305281,
+        "artId2": 510304000,
+        "artId3": 350301809,
+        "artId4": 650216515,
+        "artId5": 630220005,
         "groupId": 1014,
         "id": 30528,
     },
@@ -188,9 +214,243 @@
     "initialType": "ATTACK",
     "magiaCharaNo": 305200,
     "magiaId": 30525,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690206105,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690208800,
+                "effectCode": "CURSE",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206105,
+            "artId2": 690208800,
+            "groupId": 1119,
+            "id": 113001,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690219008,
+                "effectCode": "PROVOKE",
+                "growPoint": 0,
+                "probability": 650,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "art2": {
+                "artId": 390201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690219008,
+            "artId2": 390201105,
+            "eventArt1": {
+                "artId": 1141004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1141_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1141004,
+            "eventDescription": "＆「憩いのアイス」追加獲得数＋15",
+            "eventId": 1141,
+            "groupId": 1219,
+            "id": 165501,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 310201806,
+                "effectCode": "BLAST",
+                "effectValue": 400,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 310201304,
+                "effectCode": "DAMAGE",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 310201806,
+            "artId2": 310201304,
+            "groupId": 1092,
+            "id": 146501,
+            "intervalTurn": 6,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 310201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 610206607,
+                "effectCode": "DEFENSE_IGNORED",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 400,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 310201104,
+            "artId2": 610206607,
+            "groupId": 1085,
+            "id": 143801,
+            "intervalTurn": 5,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 305200,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690206104,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 690208800,
+                "effectCode": "CURSE",
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "artId1": 690206104,
+            "artId2": 690208800,
+            "groupId": 1119,
+            "id": 113000,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 690219006,
+                "effectCode": "PROVOKE",
+                "growPoint": 0,
+                "probability": 550,
+                "targetId": "SELF",
+                "verbCode": "IGNORE"
+            },
+            "art2": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 690219006,
+            "artId2": 390201104,
+            "eventArt1": {
+                "artId": 1141003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1141_KEY",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1141003,
+            "eventDescription": "＆「憩いのアイス」追加獲得数＋5",
+            "eventId": 1141,
+            "groupId": 1219,
+            "id": 165500,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 310201805,
+                "effectCode": "BLAST",
+                "effectValue": 350,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 310201303,
+                "effectCode": "DAMAGE",
+                "effectValue": 150,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 310201805,
+            "artId2": 310201303,
+            "groupId": 1092,
+            "id": 146500,
+            "intervalTurn": 7,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 310201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 610206606,
+                "effectCode": "DEFENSE_IGNORED",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 350,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 310201103,
+            "artId2": 610206606,
+            "groupId": 1085,
+            "id": 143800,
+            "intervalTurn": 6,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1000,
     "rateGainMpDef": 800,
