--- na/30535	2017-03-17 18:39:13.000000000 +0800
+++ jp/30535	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 100000106,
-            "effectValue": 7000,
+            "effectValue": 11200,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -153,7 +153,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190305381,
-            "effectValue": 13200,
+            "effectValue": 21120,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -196,9 +196,293 @@
     "initialType": "HEAL",
     "magiaCharaNo": 305300,
     "magiaId": 30535,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 690210403,
+                "effectCode": "CURSE",
+                "effectValue": 150,
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 200,
+                "targetId": "SELF",
+                "verbCode": "ENCHANT"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690207703,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 40,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690210403,
+            "artId2": 690206103,
+            "artId3": 690207703,
+            "groupId": 1150,
+            "id": 131301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201206,
+                "effectCode": "DEFENSE",
+                "effectValue": 450,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201904,
+                "effectCode": "MP_GAIN",
+                "effectValue": 125,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201206,
+            "artId2": 390201904,
+            "groupId": 1086,
+            "id": 182301,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 630206204,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 330201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 630206204,
+            "artId2": 330201204,
+            "eventArt1": {
+                "artId": 1182004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1182_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1182004,
+            "eventDescription": "＆「想いを記すもの」追加獲得数＋15",
+            "eventId": 1182,
+            "groupId": 1120,
+            "id": 173601,
+            "intervalTurn": 5,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 630207705,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 60,
+                "enableTurn": 3,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201105,
+            "artId2": 330201204,
+            "artId3": 630207705,
+            "groupId": 1085,
+            "id": 145701,
+            "intervalTurn": 8,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 305300,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 690210402,
+                "effectCode": "CURSE",
+                "effectValue": 150,
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 150,
+                "targetId": "SELF",
+                "verbCode": "ENCHANT"
+            },
+            "art2": {
+                "artId": 690206102,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 690207702,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 30,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 690210402,
+            "artId2": 690206102,
+            "artId3": 690207702,
+            "groupId": 1150,
+            "id": 131300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201205,
+                "effectCode": "DEFENSE",
+                "effectValue": 375,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201903,
+                "effectCode": "MP_GAIN",
+                "effectValue": 100,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201205,
+            "artId2": 390201903,
+            "groupId": 1086,
+            "id": 182300,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 630206203,
+                "effectCode": "DAMAGE_DOWN",
+                "effectValue": 150,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art2": {
+                "artId": 330201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 630206203,
+            "artId2": 330201203,
+            "eventArt1": {
+                "artId": 1182003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1182_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1182003,
+            "eventDescription": "＆「想いを記すもの」追加獲得数＋5",
+            "eventId": 1182,
+            "groupId": 1120,
+            "id": 173600,
+            "intervalTurn": 6,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 330201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 330201203,
+                "effectCode": "DEFENSE",
+                "effectValue": 225,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art3": {
+                "artId": 630207704,
+                "effectCode": "AUTO_HEAL",
+                "effectValue": 50,
+                "enableTurn": 3,
+                "growPoint": 10,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "artId1": 330201104,
+            "artId2": 330201203,
+            "artId3": 630207704,
+            "groupId": 1085,
+            "id": 145700,
+            "intervalTurn": 9,
+            "messageId": 70,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 1000,
     "rateGainMpDef": 1200,
