--- na/30535	2017-03-17 18:39:13.000000000 +0800
+++ jp/30535	2017-03-17 18:39:13.000000000 +0800
@@ -70,7 +70,7 @@
     "cardMagia": {
         "art1": {
             "artId": 100000106,
-            "effectValue": 7000,
+            "effectValue": 11200,
             "growPoint": 100,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
@@ -153,7 +153,7 @@
     "doppelCardMagia": {
         "art1": {
             "artId": 190305381,
-            "effectValue": 13200,
+            "effectValue": 21120,
             "growPoint": 220,
             "targetId": "TARGET",
             "verbCode": "ATTACK"
