--- na/40215	2017-03-17 18:39:13.000000000 +0800
+++ jp/40215	2017-03-17 18:39:13.000000000 +0800
@@ -182,9 +182,321 @@
     "initialType": "BALANCE",
     "magiaCharaNo": 402100,
     "magiaId": 40215,
-    "maxPieceSkillList": [],
+    "maxPieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 390201205,
+                "effectCode": "DEFENSE",
+                "effectValue": 375,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201104,
+            "artId2": 690206103,
+            "artId3": 390201205,
+            "eventArt1": {
+                "artId": 1181004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1181_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1181004,
+            "eventDescription": "＆「初夢祈願」追加獲得数＋15",
+            "eventId": 1181,
+            "groupId": 1085,
+            "id": 184601,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201804,
+                "effectCode": "BLAST",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201604,
+                "effectCode": "ACCEL",
+                "effectValue": 175,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201804,
+            "artId2": 390201604,
+            "eventArt1": {
+                "artId": 1158004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158004,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋15",
+            "eventId": 1158,
+            "groupId": 1092,
+            "id": 151001,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510006000,
+                "effectCode": "BAN_MAGIA",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410002007,
+                "effectCode": "ATTACK",
+                "effectValue": 350,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510006000,
+            "artId2": 410002007,
+            "eventArt1": {
+                "artId": 1092004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_DAILYTOWER_1092_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1092004,
+            "eventDescription": "＆「一度きりの時間」追加獲得数＋15",
+            "eventId": 1092,
+            "groupId": 1113,
+            "id": 147601,
+            "intervalTurn": 4,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 200200800,
+                "effectCode": "DEBUFF",
+                "growPoint": 0,
+                "targetId": "SELF",
+                "verbCode": "REVOKE"
+            },
+            "art2": {
+                "artId": 330201105,
+                "effectCode": "ATTACK",
+                "effectValue": 250,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 200200800,
+            "artId2": 330201105,
+            "eventArt1": {
+                "artId": 1158004,
+                "effectCode": "DROPADD",
+                "effectValue": 15000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158004,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋15",
+            "eventId": 1158,
+            "groupId": 1082,
+            "id": 150901,
+            "intervalTurn": 8,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "miniCharaNo": 402100,
-    "pieceSkillList": [],
+    "pieceSkillList": [
+        {
+            "art1": {
+                "artId": 390201103,
+                "effectCode": "ATTACK",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 690206103,
+                "effectCode": "DAMAGE_UP",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "CONDITION_GOOD"
+            },
+            "art3": {
+                "artId": 390201204,
+                "effectCode": "DEFENSE",
+                "effectValue": 300,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201103,
+            "artId2": 690206103,
+            "artId3": 390201204,
+            "eventArt1": {
+                "artId": 1181003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1181_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1181003,
+            "eventDescription": "＆「初夢祈願」追加獲得数＋5",
+            "eventId": 1181,
+            "groupId": 1085,
+            "id": 184600,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 390201803,
+                "effectCode": "BLAST",
+                "effectValue": 250,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "art2": {
+                "artId": 390201603,
+                "effectCode": "ACCEL",
+                "effectValue": 150,
+                "growPoint": 25,
+                "probability": 1000,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 390201803,
+            "artId2": 390201603,
+            "eventArt1": {
+                "artId": 1158003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158003,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋5",
+            "eventId": 1158,
+            "groupId": 1092,
+            "id": 151000,
+            "skillDisplayType": "MEMORIA",
+            "type": "ABILITY"
+        },
+        {
+            "art1": {
+                "artId": 510006000,
+                "effectCode": "BAN_MAGIA",
+                "enableTurn": 1,
+                "growPoint": 0,
+                "probability": 1000,
+                "targetId": "TARGET",
+                "verbCode": "CONDITION_BAD"
+            },
+            "art2": {
+                "artId": 410002006,
+                "effectCode": "ATTACK",
+                "effectValue": 300,
+                "enableTurn": 1,
+                "growPoint": 25,
+                "targetId": "TARGET",
+                "verbCode": "DEBUFF"
+            },
+            "artId1": 510006000,
+            "artId2": 410002006,
+            "eventArt1": {
+                "artId": 1092003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_DAILYTOWER_1092_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1092003,
+            "eventDescription": "＆「一度きりの時間」追加獲得数＋5",
+            "eventId": 1092,
+            "groupId": 1113,
+            "id": 147600,
+            "intervalTurn": 5,
+            "messageId": 72,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        },
+        {
+            "art1": {
+                "artId": 200200800,
+                "effectCode": "DEBUFF",
+                "growPoint": 0,
+                "targetId": "SELF",
+                "verbCode": "REVOKE"
+            },
+            "art2": {
+                "artId": 330201104,
+                "effectCode": "ATTACK",
+                "effectValue": 200,
+                "enableTurn": 3,
+                "growPoint": 25,
+                "targetId": "SELF",
+                "verbCode": "BUFF"
+            },
+            "artId1": 200200800,
+            "artId2": 330201104,
+            "eventArt1": {
+                "artId": 1158003,
+                "effectCode": "DROPADD",
+                "effectValue": 5000,
+                "genericValue": "EVENT_SINGLERAID_1158_EXCHANGE_1",
+                "verbCode": "OTHER"
+            },
+            "eventArtId1": 1158003,
+            "eventDescription": "＆「天使のメダル」追加獲得数＋5",
+            "eventId": 1158,
+            "groupId": 1082,
+            "id": 150900,
+            "intervalTurn": 9,
+            "skillDisplayType": "MEMORIA",
+            "type": "SKILL"
+        }
+    ],
     "rank": "RANK_5",
     "rateGainMpAtk": 900,
     "rateGainMpDef": 900,
